cluster-setup/base/performance/performance_profile.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: performance.openshift.io/v1alpha1
 kind: PerformanceProfile
 metadata:
-  name: ci
+  name: performance
```

Comment:
This one is still not merged https://github.com/openshift/machine-config-operator/pull/1356, so can you change it back to shorter name

Reply:
ah yes, that was the reason. Thanks for the heads up!